exercise-src/react-native/18-maps/01-solution/src/env.d.ts

```diff
@@ -5,8 +5,8 @@ declare global {
     interface ProcessEnv {
       readonly PMO_API: string
       readonly PMO_ASSETS: string
-      readonly GOOGLE_MAPS_API_KEY: string
       readonly GOOGLE_OAUTH_CLIENT_ID: string
+      readonly GOOGLE_MAPS_API_KEY: string
```

Comment:
Same here, why this change? Alphabetical order is always nice for quickly scanning.

Reply:
For trainings, I also sorting things in the order they're introduced to the audience. Besides making it clearer where they will put things or where to look in solutions, it also improves the quality an cleanliness of diffs (in general, but even moreso with the academy diff algorithm). (The diffing doesn't apply to this line, it explains the changes in several files in the last PR.)